### PR TITLE
Fixed merge bugs

### PR DESCRIPTION
**Pull request by patw0929**
Hey @housseindjirdeh , really sorry for the late reply. 
I found you've merged #49, however there have several eslint errors & javascript errors. 
So I open this PR, trying to solve them. Please help to review this again, thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +56,8 @@
     "redux-logger": "^2.7.4",
     "redux-persist": "^4.3.1",
     "redux-persist-transform-encrypt": "^1.0.2",
+    "remote-redux-devtools": "^0.5.12",
```
Comment (housseindjirdeh) [inline package.json]: Although the linter is yelling to have these dependencies - this really shouldn't be in `dep` and should be in `devDep`. We can do `// eslint-disable-line import/no-extraneous-dependencies` as a workaround where they're being imported

File: src/components/search-bar.component.ios.js [hunk 2]
```diff
@@ -12,6 +12,13 @@ export class SearchBar extends Component {
     onSearchButtonPress: PropTypes.func.isRequired,
   };
 
+  static defaultProps = {
```
Comment (housseindjirdeh) [inline src/components/search-bar.component.ios.js]: Just realized you're using the `proptypes` package to define props in these components. The rest of the application uses flow so I think it makes sense to do the same here. You can do this by adding a simple `Props` object like so:

```
type Props = {
  textColor: string,
  textFieldBackgroundColor: string,
  showsCancelButton: bool,
  placeholder: string,
  onFocus: Function,
  onCancelButtonPress: Function,
  onSearchButtonPress: Function,
};
```

and then adding it as a type:

```
export const Searchbar = ({
  textColor,
  textFieldBackgroundColor,
  showsCancelButton,
  placeholder,
  onFocus,
  onCancelButtonPress,
  onSearchButtonPress,
}: Props) => {
```

This should save us the need to define defaultProps as well.

Also each of the components in the `components` folder are stateless functional. Although everyone has their own preferences, I think it makes sense to follow the same format here as well. Will prefer if you can make `search-bar.ios` and `search-bar.android` the same if that's okay :)

File: package.json (hunk 1 above)
Comment (patw0929) [inline package.json]: 👍

File: src/components/search-bar.component.ios.js (hunk 2 above)
Comment (patw0929) [inline src/components/search-bar.component.ios.js]: Cool, I am not familiar with flow. Looks better.
Fixed in search-bar ios & android.

**Comment by housseindjirdeh** [comment]
Thanks a million once again <3 - it looks good now! Please me know if you're done with updates and I'll merge this in

**Comment by patw0929** [comment]
Done! 🚀🚀🚀
Thanks!

**Pull request merged**